src/netaddress.cpp

```diff
@@ -643,13 +643,16 @@ bool CService::GetSockAddr(struct sockaddr* paddr, socklen_t *addrlen) const
     return false;
 }
 
+/**
+ * @returns An identifier unique to this service's address and port number.
+ */
 std::vector<unsigned char> CService::GetKey() const
 {
      std::vector<unsigned char> vKey;
      vKey.resize(18);
      memcpy(vKey.data(), ip, 16);
-     vKey[16] = port / 0x100;
-     vKey[17] = port & 0x0FF;
+     vKey[16] = port / 0x100; // first (most significant) byte of our port
```

Comment:
In commit "docs: netaddress: Add CService::GetKey docs" (b3697888fcff02eccc84521c2b64e0ae21a8236a)

Would drop first/second and just say most significant/least significant or high/low.

With first/second it sounds like this is making an assumption about the byte order.